static/app/bootstrap/bootstrapRequests.tsx

```diff
@@ -52,9 +46,23 @@ export function useBootstrapOrganizationQuery(orgSlug: string | null) {
       });
     }
     if (organizationQuery.error) {
-      OrganizationStore.onFetchOrgError(organizationQuery.error as any);
+      if (
+        organizationQuery.error instanceof RequestError &&
+        // By default, `react-query` will refetch stale queries on window focus.
+        // This can lead to many queries that hit Sentry's API rate limit. In the case
+        // of the organization details endpoint, an error means that the OrganizationStore
+        // will not have an organization, which leads to `useOrganization` throwning because
+        // organization is undefined. This ultimately leads to a "crash" for the user
+        // (e.g. blank page w/ an error message).
+        !(
+          organizationQuery.error.status === 429 &&
+          orgSlug === OrganizationStore.state.organization?.slug
```

Comment:
Maybe there's an argument to be made to avoid `null`-ing out the org (provided the org is the same) in the store for all errors as it leads to a pretty shitty UX.